dashboard/test/ui/features/hamburger.feature

```diff
@@ -0,0 +1,150 @@
+Feature: Hamburger dropdown
+
+  Scenario: Signed out user in English should not see hamburger on desktop
+    Given I am on "http://code.org/"
+    And I set the language cookie
+    Then I wait until element "#hamburger-icon" is not visible
```

Comment:
I think in this case, wait until there's something you can see (e.g. logo) and then just make sure this icon isn't visible immediately.  It doesn't make sense to wait for it to not be visible :)